package_directory.nim

```diff
@@ -611,32 +614,23 @@ proc fetch_github_versions(pkg: Pkg, owner_repo_name: string) {.async.} =
 
     pkg["github_latest_version_time"] = newJString ""
 
-proc fetch_github_repository_stats(sorting = "updated", pagenum = 1,
-    limit = 200, initial_date: DateTime):
-    Future[seq[JsonNode]] {.async.} =
-  ## Fetch projects on GitHub
-  let date = initial_date.format("yyyy-MM-dd")
-  let q = github_repository_search_tpl % [date, $limit, sorting, $pagenum]
-  log_info "Searching GH repos: '$#'" % q
-  let query_res = await getGHJson(q)
-  if sorting == "updated":
-    return query_res["items"].elems.sortedByIt(it["updated_at"].str).reversed()
-  return query_res["items"].elems
 
-proc github_trending_packages(request: Request, pkgs: Pkgs): Future[seq[
-    JsonNode]] {.async.} =
-  ## Trending GitHub packages
-  # TODO: Dom: merge this into the procedure above ^
+proc fetch_trending_packages(request: Request, pkgs: Pkgs): Future[seq[JsonNode]] {.async.} =
 
   if volatile_cache_github_trending_last_update_time +
       github_caching_time > epochTime().int:
     return volatile_cache_github_trending
 
+  let date = utc(getTime() - 14.days).format("yyyy-MM-dd")
+  let q = github_repository_search_tpl % [date, "20", "updated", "1"]
+  log_info "Searching GH repos: '$#'" % q
+  let query_res = await getGHJson(q)
+  # NOTE: getGHJson should have descriptions in it???
```

Comment:
uh?

Reply:
What's weird here, the comment? I think it might have been from my notes while reading though the codebase - it doesn't look it's present in the final diff.